soc/arm/infineon_cat1/psoc6/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2021 Cypress Semiconductor Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_sources(soc.c)
+zephyr_include_directories(.)
+
+include(${CMAKE_CURRENT_SOURCE_DIR}/CMakeLists.psoc6_02.txt)
```

Comment:
I am not sure if this is good practice. @tejlmand

Reply:
i have die family specific CMakeLists.psoc6_xx.txt files (it was grouped in such way for easy maintains/update in future)
CMakeLists.psoc6_01.txt
CMakeLists.psoc6_02.txt (In this PR we add only this one)
CMakeLists.psoc6_03.txt
CMakeLists.psoc6_04.txt

So you suggest to put all content from CMakeLists.psoc6_xx.txt to CMakeLists.txt to get rid of "include".
Is it because consistency reason, or there are some specific of using "include" in Cmake at all...?